tangent/grads.py

```diff
@@ -308,8 +308,19 @@ def adet(z, x):
 
   See  Jacobi's formula: https://en.wikipedia.org/wiki/Jacobi%27s_formula
   """
-  adjugate = np.linalg.det(x) * np.linalg.pinv(x)
+  adjugate = numpy.linalg.det(x) * numpy.linalg.pinv(x)
   d[x] = adjugate.T
```

Comment:
This should be `numpy.transpose(adjugate)` (so that higher-order gradients can be taken).

Reply:
I think I have fixed this.